webpack.config.js

```diff
@@ -1,10 +1,19 @@
 const path = require('path');
+const webpack = require('webpack');
+const CleanWebpackPlugin = require('clean-webpack-plugin')
 
 module.exports = {
-	entry: 'src/root-application/root-application.js',
+	entry: {
+		'root-application': 'src/root-application/root-application.js',
+		'common-dependencies': [
```

Comment:
We don't want multiple versions of angular, so we put the library itself into a common dependencies bundle.